data.base.api/src/main/java/ru/tinkoff/qa/neptune/data/base/api/queries/JDOPersistenceManagerByPersistableClass.java

```diff
@@ -0,0 +1,30 @@
+package ru.tinkoff.qa.neptune.data.base.api.queries;
+
+import org.datanucleus.api.jdo.JDOPersistenceManager;
+import ru.tinkoff.qa.neptune.data.base.api.DataBaseStepContext;
+import ru.tinkoff.qa.neptune.data.base.api.PersistableObject;
+
+import java.util.function.Function;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static ru.tinkoff.qa.neptune.data.base.api.ConnectionToUse.ConnectionDataReader.getConnection;
+
+class JDOPersistenceManagerByPersistableClass<T extends PersistableObject> implements Function<DataBaseStepContext, JDOPersistenceManager> {
+
+    private final Class<T> clazz;
+
+    private JDOPersistenceManagerByPersistableClass(Class<T> clazz) {
+        checkNotNull(clazz);
```

Comment:
Should error msg be described for that clazz?

Reply:
It is not necessary here